### PR TITLE
quad-control: add target attitude and enable yaw pid calculations

### PR DESCRIPTION
**Pull request by niewim19**
## Description

 - add target attitude parsing from `etc/quad.conf`
 - add target attitude usage in pid calculations

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## How Has This Been Tested?

- [ ] Already covered by automatic testing.
- [ ] New test added: (add PR link here).
- [x] Tested by hand on: zturn drone

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing linter checks and tests passed.
- [ ] My changes generate no new compilation warnings for any of the targets.

## Special treatment

- [ ] This PR needs additional PRs to work (list the PRs, preferably in merge-order).
- [ ] I will merge this PR by myself when appropriate.

### CODE REVIEW

File: quadcontrol/control.h [hunk 1]
```diff
@@ -30,6 +30,13 @@ typedef struct {
 } flight_takeoff_t;
 
 
+typedef struct {
+	float yaw;   /* yaw angle in milliradians in range (-PI, PI] */
+	float pitch; /* pitch angle in milliradians in range (-PI/2, PI/2] */
+	float roll;  /* roll angle in milliradians in range (-PI/2, PI/2] */
+} target_attitude_t;
```
Comment (HBuczynski) [inline quadcontrol/control.h:37]:
```suggestion
} quad_att_t;
```
or
```suggestion
} att_t;
```

File: quadcontrol/control.c [hunk 2]
```diff
@@ -400,13 +446,23 @@ static int quad_configRead(void)
...
+				printf("pids\n");
 				break;
 			}
 		}
 
 		/* @THROTTLE - define throttle min & max values */
 		if ((line[0] == '@') && (strcmp(&line[1], "THROTTLE\n") == 0)) {
 			err = quad_throttleParse(file);
+			if (err < 0) {
+				printf("throttle\n");
```
Comment (HBuczynski) [inline quadcontrol/control.c:458]:
```suggestion
				fprintf(stderr, "quad-control: cannot parse throttle parameters\n");
```
Suggestion applies to the rest of the conditions in `quad_pidParse` function.

File: quadcontrol/control.c [hunk 3]
```diff
@@ -41,10 +41,12 @@
 
 #define ANGLE_THRESHOLD (M_PI / 6)
 #define RAD2DEG         ((float)180.0 / M_PI)
+#define DEG2RAD         0.0174532925f
 
 
 struct {
 	pid_ctx_t pids[PID_NUMBERS];
+	target_attitude_t targetAtt;
```
Comment (HBuczynski) [inline quadcontrol/control.c:49]: is `targetAtt` used only for `TEST_ATTITUDE`?

Comment (niewim19) [reply quadcontrol/control.c:49]: Currently yes, it is.

**Pull request merged**